TombEngine/Game/camera.cpp

```diff
@@ -405,6 +405,16 @@ void MoveObjCamera(GameVector* ideal, ItemInfo* camSlotId, int camMeshId, ItemIn
 	}
 }
 
+void FixedCameraRefresh(short camNumber)
```

Comment:
Maybe rename function to just "CameraRefresh"?

Reply:
Not this case, I put the "Fixed" because I wanted make a distinction that this is updated for the fixed cameras.

levelCameras (the objects you put in the level) has some data that the Main Camera reads to positionate (and do its fixedCamera rutine), but it needs that levelCamera object. The other cameras rutines (chase, look, combat, etc) do not need those levelCameras, so this Refresh function actually do nothing to them.

Is, for what it's, to force a camera refresh in the FixedCamera rutine.

Reply:
If you're going to change the function's name, it should be "RefreshFixedCamera", so it follows the "DoSomething" kind of naming which is already used, and more natural for function names, rather than something along the lines of "SomeAction".

Reply:
Yeah, that can be done.

Reply:
Done